src/app/shared/dialogs/dialogs-form.component.ts

```diff
@@ -23,7 +23,22 @@ export class DialogsFormComponent {
   public validation: any;
   public message: string;
   public modalForm: FormGroup;
+  readonly dbName = 'nations';
```

Comment:
this is not supposed to be readonly field as dialog-form could work with any other modal